### PR TITLE
ACP-773: Stripe documentation mvp

### PR DESCRIPTION
**Pull request by danielsantos-spryker**
## PR Description

TBD

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by matweew — CHANGES_REQUESTED** [review]

**Review by matweew — DISMISSED** [review]
From technical point of view it's good.

**Review by stereomon — APPROVED** [review]

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/configure-stripe.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+---
+This document describes how to integrate [Stripe](/docs/pbc/all/payment-service-provider/{{page.version}}/base-shop/third-party-integrations/stripe/stripe.html) into a Spryker shop.
+
+## Prerequisites
+
+Before integrating Stripe, ensure the following prerequisites are met:
+
+- Make sure your project is ACP-enabled. See [App Composition Platform installation](/docs/acp/user/app-composition-platform-installation.html) for details.
+- Make sure you have a Stripe connected account to Spryker platform, if you are unsure how to do it, please contact Spryker support.
```
[the remark this answers is not in the document]
Comment (danielsantos-spryker) [reply docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/configure-stripe.md:21]: - Make sure your project is ACP-enabled. See [App Composition Platform installation](/docs/acp/user/app-composition-platform-installation.html) for details.

This part might not be needed, you wouldn't be able to configure without install first and to install your project would have to be ACP-enabled

[the remark this answers is not in the document]
Comment (danielsantos-spryker) [reply docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/configure-stripe.md:21]: - Make sure you have a Stripe connected account to Spryker platform, if you are unsure how to do it, please contact Spryker support.

For configuration you will need an account ID from Stripe

File: docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/configure-stripe.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+   {% endinfo_block %}
+
+## 2. Configure Stripe
+
+1. Go to your store's Back Office, to the Stripe app details page.
+2. In the top right corner of the Stripe app details page, click **Configure**.
+3. On the Stripe app details page, fill in the account ID field in the **Configuration** section.
+4. Select **Stripe Environment Mode**.
+5. Enter your *Shop Name*. This name will be displayed on **Payment** page as a merchant label for whom to pay:
```
[the remark this answers is not in the document]
Comment (danielsantos-spryker) [reply docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/configure-stripe.md:45]: Screenshot 2024-01-16 at 10 19 44

Please check

**Pull request merged**